SoftMemoryLimit/MemoryChecker.php

```diff
@@ -104,8 +104,11 @@ protected function parseMemoryLimit(string $memoryLimit): int {
 
    // No Cloning or Serializing.
    private function __clone() {}
-   private function __wakeup() {}
+   public function __wakeup(): void {
```

Comment:
Wouldn't you want to do this on __sleep()? Seems like if it's trying to unserialize, we've already caused the problem.

Reply:
I left the `__wakeup()` override, even though it might be unnecessary, cause I don't know why exactly it was implemented like that in the first place and it seems harmless.